test/javascripts/acceptance/question-answer-test.js

```diff
@@ -725,6 +725,34 @@ acceptance("Discourse Question Answer - logged in user", function (needs) {
     );
   });
 
+  test("receving post commented message as soon as the message is created by user", async function (assert) {
+    updateCurrentUser({ id: 12345 });
+
+    await visit("/t/280");
+
+    publishToMessageBus("/topic/280", {
+      type: "qa_post_commented",
+      id: topicResponse.post_stream.posts[0].id,
+      comments_count: 2,
+      user_id: 12345,
```

Comment:
```suggestion
```

I have a feeling this does not need to be in the payload.

Reply:
Good catch! Fixed.